cloud/datasync/src/SyncServer.ts

```diff
@@ -4,6 +4,16 @@ import {setupGlobalHandlers, setupHandlers} from './crud-handlers/HandlerMapper'
 import SyncDataSetOptions from './options/SyncDatasetOptions';
 import SyncOptions from './options/SyncGlobalOptions';
 import SyncApi from './SyncApi';
+import {BunyanLogger,LOG_LEVEL} from '../../logger-cloud/src/index';
+//import {BunyanLogger,LOG_LEVEL} from '@raincatcher/logger-cloud'; this don't work
```

Comment:
run bootstrap and it should work, we can't keep references to outside `datasync`